x-pack/legacy/plugins/uptime/public/components/functional/location_map/embeddables/embedded_map.tsx

```diff
@@ -49,15 +49,15 @@ export const EmbeddedMap = React.memo(({ upPoints, downPoints }: EmbeddedMapProp
   const embeddableRoot: React.RefObject<HTMLDivElement> = useRef<HTMLDivElement>(null);
   const factory = start.getEmbeddableFactory(MAP_SAVED_OBJECT_TYPE);
 
-  const input = {
+  const input: MapEmbeddableInput = {
     id: uuid.v4(),
     filters: [],
     hidePanelTitles: true,
     refreshConfig: {
       value: 0,
       pause: false,
     },
-    viewMode: 'view',
+    viewMode: ViewMode.VIEW,
```

Comment:
What is `viewMode`? This is not part of MapEmbeddableInput type defined in map_embeddable

Reply:
Part of the base EmbeddableInput interface.  It's what dashboard passes down to tell Embeddables whether they are in view or edit mode.